data/simple-icons.json

```
@@ -571,7 +572,7 @@
 	{
 		"title": "Alby",
 		"hex": "FFDF6F",
-		"source": "https://github.com/getAlby/media/blob/c24fee4a3f76d6cd000343a972f10590d3913b25/Alby-logo-icons/Alby-logo-head/alby.svg",
+		"source": "https://github.com/getAlby/media/blob/79cf8fc9a95e49ffa69792190d66b4b017bb80c0/Logos/Alby/Alby%20Head/alby.svg",
```

```suggestion
		"source": "https://github.com/getAlby/media/blob/21e727c8f0d6d58fde617815e5fe84b2daa1c401/Logos/Alby/Logo/Alby%20Logo%20-%20Monochrome.svg",
```
The logo itself has changed too, so we need to update the SVG